test/quantization/test_gptq.py

```diff
@@ -18,13 +18,15 @@
 from torchao._models.llama.tokenizer import get_tokenizer
 from torchao.quantization import Int4WeightOnlyConfig, quantize_
 from torchao.quantization.utils import compute_error
+from torchao.utils import auto_detect_device
 
 torch.manual_seed(0)
 
+_DEVICE = auto_detect_device()
```

Comment:
auto_detect_device seems to be changing what we want to test, I think previous we only want to test on CUDA, can you preserve this?

Reply:
We have refine the auto_detect_device functions and cpu will not be included.